Libraries/Components/Touchable/TouchableOpacity.js

```diff
@@ -245,13 +246,15 @@ class TouchableOpacity extends React.Component<Props, State> {
         nativeID={this.props.nativeID}
         testID={this.props.testID}
         onLayout={this.props.onLayout}
-        nextFocusDown={this.props.nextFocusDown}
-        nextFocusForward={this.props.nextFocusForward}
-        nextFocusLeft={this.props.nextFocusLeft}
-        nextFocusRight={this.props.nextFocusRight}
-        nextFocusUp={this.props.nextFocusUp}
+        nextFocusDown={tagForComponentOrHandle(this.props.nextFocusDown)}
```

Comment:
Could be nice, at the time I was also thinking this could be done via a11y labels / ids, as is recommended for when one wants to look up components for tests. That'd also promote better a11y for lib users.